cumulusci/tasks/tests/test_metadeploy.py

```diff
@@ -145,7 +144,7 @@ def test_run_task(self):
 
         responses.add(
             "GET",
-            "https://metadeploy/products?repo_url=EXISTING_REPO",
+            "https://metadeploy/products?repo_url=https%3A%2F%2Fgithub.com%2FTestOwner%2FTestRepo",
```

Comment:
@jstvz I don't think it's expecting the result to be html escaped. Although I don't think this is a bad time to add an html.escape() addition to our repo URL https://github.com/SFDO-Tooling/CumulusCI/pull/3200/files#diff-97be17eb03341832c29a8f929a2358faf7a3157182956ce96eb7170cfcdea1e4L242 let me know what you think.

Reply:
It's not clear because of the old test value for `repo_url`, but MetaDeploy is expecting a URL encoded string as the query parameter. Updated the test to match.